src/index.html

```diff
@@ -1,13 +1,58 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="./styles/index.css">
+  </head>
+  <body>
+    <article class="product" data-qa="card">
+      <img
+       src="./images/imac.jpeg"
+       alt="photo of Apple A1419 iMac"
+       class="product__image"
+      >
+
+      <h3 class="product__disc">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </h3>
+
+      <h4 class="product__code">
+        Product code: 195434
+      </h4>
+
+      <div class="review">
+        <div class="stars stars--4">
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+        </div>
+
+        <p class="review__num">
+        Reviews: 5
+        </p>
+
+      </div>
+      <div class="product__price">
+        <p class="product__text">Price:</p>
+        <p class="product__number">$2,199</p>
+      </div>
+
+      <a
+        href="#"
+        class="product__buy"
+        data-qa="hover"
+      >
+        BUY
```

Comment:
It is not recommended to hardcode uppercase. JFYI.

Reply:
agreed